### ISSUE TITLE
MultiPage and Spacer

### ISSUE THREAD

**Issue by peterstojanowski** [DavBfr/dart_pdf#338]
**Is your feature request related to a problem? Please describe.**

Hi @DavBfr, I have a requirement to build a pdf with multiple pages where some widgets are aligned to the bottom of the last page. I've tried using the `Spacer` widget but it seems it works only with single `Page` widgets.

**Describe the solution you'd like**

I would like to see `Spacer` widget working the same way as embedded in `Page` widget.

**Describe alternatives you've considered**

An alternative would be to use the footer parameter of `MultiPage` but it would need to support variable height. The way to achieve this would be to check if we are on the last page (using `Context`) and build a footer with additional widgets. Currently the footer has fixed height on all pages no matter if smaller on other pages.

**Additional context**

I'm happy to buy a coffee in return or sponsor development of this feature if can be delivered in relatively short amount of time ;)

Thanks

**Comment by DavBfr** [comment]
Yes, you can sponsor it, and I can do it today.

**Comment by DavBfr** [comment]
I just uploaded a new version to GitHub. You can test it using this in your `pubspec.yaml` until it's published.

```dart
dependency_overrides:
  pdf:
    git:
      url: https://github.com/DavBfr/dart_pdf
      path: pdf
```

**Comment by peterstojanowski** [comment]
Hi @DavBfr, I just checked it and it's working as expected. Thanks for implementing this so quickly.

**Comment by DavBfr** [comment]
Fixed in version 1.9.0
